$NetBSD: patch-CMakeLists.txt,v 1.4 2025/12/19 20:10:58 maya Exp $

Don't try to find the git hash.
Avoid erroring because of issues in headers (openimageio)

--- CMakeLists.txt.orig	2023-09-24 11:40:23.000000000 +0000
+++ CMakeLists.txt
@@ -59,7 +59,6 @@ else()
   set(OLIVE_COMPILE_OPTIONS
     "$<$<CONFIG:RELEASE>:-O2>"
     -Wuninitialized
-    -pedantic-errors
     -Wall
     -Wextra
     -Wno-unused-parameter
@@ -233,21 +232,6 @@ elseif(UNIX)
   list(APPEND OLIVE_LIBRARIES Qt${QT_VERSION_MAJOR}::DBus)
 endif()
 
-# Generate Git hash
-set(PROJECT_LONG_VERSION ${PROJECT_VERSION})
-if(EXISTS "${CMAKE_SOURCE_DIR}/.git")
-  find_package(Git)
-  if(GIT_FOUND)
-    execute_process(COMMAND ${GIT_EXECUTABLE} rev-parse --short=8 HEAD
-      WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-      OUTPUT_VARIABLE GIT_HASH
-      OUTPUT_STRIP_TRAILING_WHITESPACE
-    )
-
-    set(PROJECT_LONG_VERSION ${PROJECT_VERSION}-${GIT_HASH})
-  endif()
-endif()
-
 # Optional: Find Doxygen if requested
 if(BUILD_DOXYGEN)
   find_package(Doxygen)
